package.json

```diff
@@ -51,5 +51,10 @@
       "babel-runtime": "npm:babel-runtime@^5.8.24",
       "core-js": "npm:core-js@^1.1.4"
     }
+  },
+  "devDependencies": {
+    "fs-readfile-promise": "^2.0.1",
+    "nock": "^2.17.0",
```

Comment:
can be done in another PR, but we should move most of the dependencies into devDependencies.

In fact, given the app is client side basically every dependency here is a dev one.